build.sbt

```diff
@@ -311,6 +311,7 @@ lazy val remote =
     .settings(OSGi.remote)
     .settings(Protobuf.settings)
     .settings(parallelExecution in Test := false)
+    .enablePlugins(Jdk9)
```

Comment:
I added this to make it work from tests (multi-jvm):

```
  .dependsOn(
    remote % "compile->CompileJdk9",
```

Reply:
I think that's the wrong way to go about it so added a dependency on jdk-9 compilation from jdk-9-test-compilation on top of the JDK9 improvements in #28347